BDArmory.Core/BDArmorySettings.cs

```diff
@@ -43,9 +43,11 @@ public class BDArmorySettings
         [BDAPersistentSettingsField] public static bool VESSEL_RELATIVE_BULLET_CHECKS = false;
         [BDAPersistentSettingsField] public static bool AIM_ASSIST = true;
         [BDAPersistentSettingsField] public static bool DRAW_AIMERS = true;
-        [BDAPersistentSettingsField] public static bool DRAW_DEBUG_LINES = false;
+        // Debug Labels
+        [BDAPersistentSettingsField] public static bool DRAW_DEBUG_LINES = false;                   //AI/Weapon aim visualizers
         [BDAPersistentSettingsField] public static bool DRAW_DEBUG_LABELS = false;
         [BDAPersistentSettingsField] public static bool DRAW_ARMOR_LABELS = false;                 //armor only debug messages, for testing/debugging. remove/revert back to debug_labels later
+
```

Comment:
Was going to add new debug categories, but that's a bit outside of scope for this commit

Reply:
Yeah, that can be done later when there's less urgent things to do. It's a nice-to-have feature, but not necessary.